app/services/articles/feed.rb

```diff
@@ -0,0 +1,43 @@
+module Articles
+  class Feed
+    attr_reader :stories
+
+    def initialize(number_of_articles:, page:, tag: nil)
+      @number_of_articles = number_of_articles
+      @page = page
+      @tag = tag
+      @stories = published_articles_by_tag
+    end
+
+    def published_articles_by_tag
```

Comment:
You're probably planning on using this in a following PR but it seems to be called only here, so should it be private?

Reply:
Using it externally to the service now!